impl/src/tagged_trait.rs

```diff
@@ -194,11 +194,15 @@ fn externally_tagged(input: &ItemTrait) -> (TokenStream, TokenStream) {
     (serialize_impl, deserialize_impl)
 }
 
-fn internally_tagged(tag: LitStr, input: &ItemTrait) -> (TokenStream, TokenStream) {
+fn internally_tagged(tag: LitStr, default_variant: Option<LitStr>, input: &ItemTrait) -> (TokenStream, TokenStream) {
     let object = &input.ident;
     let object_name = object.to_string();
     let (_, ty_generics, _) = input.generics.split_for_impl();
     let static_registry = static_registry();
+    let default_variant_literal = match default_variant {
+        Some(ref variant) => quote!(Some(#variant)),
+        None => quote!(None),
+    };
```

Comment:
This needs to use fully qualified paths for `Option` because the words "Some" and "None" are able to mean something different in the scope where this macro is called.

```suggestion
    let default_variant_literal = match default_variant {
        Some(ref variant) => quote!(typetag::__private::Option::Some(#variant)),
        None => quote!(typetag::__private::Option::None),
    };
```